databases/week2/lesson2.sql

```diff
@@ -0,0 +1,228 @@
+--------------------Part 1: Working with tasks
+
+--Add a task with these attributes: title, description, created, updated, due_date, status_id, user_id
+INSERT INTO task (title, description, created, updated, due_date, status_id, user_id) 
+VALUES ("writing", "practice danish", now(),ADDDATE(now(), INTERVAL 1 Day),ADDDATE(now(), INTERVAL 10 Day), 3, 1 );
+
+--Change the title of a task
+UPDATE task 
+SET title = "learning Danish"
+where id = 36;
+
+--Change a task due date
+UPDATE task
+SET due_date = ADDDATE(now(), INTERVAL 5 Day)
+WHERE id = 36;
+--Change a task status
+UPDATE task
+SET status_id = 2
+WHERE id = 36;
+
+--Mark a task as complete
+UPDATE task
+SET status_id = 3
+where id = 36;
+
+--Delete a task
+DELETE FROM task
+WHERE id = 37;
+
+------------------------------------------
+-----------------------Part 2: School database
+--Create a new database containing the following tables:
+--Class: with the columns: id, name, begins (date), ends (date)
+--Student: with the columns: id, name, email, phone, class_id (foreign key)
+CREATE database School;
+use School;
+create table `Class`(
+`id` INT (10) UNSIGNED PRIMARY KEY AUTO_INCREMENT,
+`name` VARCHAR (255) NOT NULL,
+`begins` DATETIME,
+`ends` DATETIME 
```

Comment:
Indentation here and below should be updated similar to how you made in Part4 (from line 107 - 111)